support-redemptiondb/build.sbt

```diff
@@ -0,0 +1,12 @@
+
+name := "support-redemptiondb"
+description:= "packages the database cfn for the redemption code db"
+
+assemblyJarName := "dummy.jar"
```

Comment:
`dummy.jar` because there is no jar?

Reply:
yes, normally the jar is the key part (after all, it's an SBT build) and the riffRaffArtifactResources are extras on the side.  but this time it's backwards.
I should mentioned that an alternative approach is the approach I used in https://github.com/guardian/support-frontend/tree/master/support-frontend-static which was just get people to run a script locally to deploy.  But generally I dislike people deploying or building PROD stuff from their laptops as it's less controlled.